orchagent/orchdaemon.cpp

```diff
@@ -275,6 +280,56 @@ bool OrchDaemon::init()
 
     gMacsecOrch = new MACsecOrch(m_applDb, m_stateDb, macsec_app_tables, gPortsOrch);
   
+    Table m_switchTable(stateDbSwitchTable.first, stateDbSwitchTable.second);
+    // Check if SAI is capable of handling TPID config and store result in StateDB switch capability table
+    {
```

Comment:
Please move this section to switch init and make this part of switchorch class.

Reply:
Changed per suggestion in my new commit.